src/api/certificateAuthority.js

```diff
@@ -1,7 +1,7 @@
 import Q from "q";
 import logger from "winston";
 import pem from "pem";
-import { Keystore, Certificate } from "../model/keystore";
+import { KeystoreModelAPI, CertificateAPI } from "../model/keystore";
```

Comment:
@BMartinos I wasn't actually sure which name to use; in restart.js, there is a variable named 'KeystoreAPI' (`import * as KeystoreAPI from "../api/keystore";`), and the keystore model needs to be imported here as well (so I used a different name to avoid the conflict). Another possible solution could be use KeystoreAPI for the model, and rename the import reference, what do u think?